README.md

```diff
@@ -228,6 +228,10 @@ deploy workloads.
 ### Cluster-wide proxy 
 WMCO does not support adding Windows workloads using a [cluster-wide proxy](https://docs.openshift.com/container-platform/latest/networking/enable-cluster-wide-proxy.html) config for the OpenShift Container Platform. WMCO will not be able to automatically route proxy connections for Windows workloads.
 
+### Accessing secure registries
+WMCO does not support pulling images from secure private registries on the Windows nodes. 
```

Comment:
This sentence could be restructured like so:
`Windows nodes managed by WMCO do not support pulling container images from secure private registries.`

I'd do this as the subject is the `Windows nodes managed by WMCO`, as WMCO itself is not doing the pulling.
I also replaced `images` with `container images`, to be explicit in differentiating it from the VM image referred to in the next sentence.

Reply:
Makes sense, thanks for the suggested wording. Made the changes.